releasenotes/notes/full_prec_sympy-aeee8210091ef20f.yaml

```diff
@@ -0,0 +1,13 @@
+---
+other:
+  - |
+    The string cast for :class:`qiskit.circuit.Parameter` does not have full precision
+    anymore. This removes the trailing 0s when printing parameters that are bound to floats.
+    This has consequence down to, for example, QASM serialization and circuit text drawer.
+
+    .. code-block:: python
+
+      >> from qiskit.circuit import Parameter
+      >> x = Parameter('x')
+      >> str(x.bind({x:0.5}))
+     '0.5'   // instead of '0.500000000000000'
```

Comment:
```suggestion
      >>> from qiskit.circuit import Parameter
      >>> x = Parameter('x')
      >>> str(x.bind({x:0.5}))
      '0.5'   // instead of '0.500000000000000'
```